website/docs/concepts/event-bridge/event-bus.md

```diff
@@ -6,343 +6,125 @@ sidebar_position: 1
+## Default Bus
 
-```ts
-interface UserEvent
-  extends functionless.EventBusRuleInput<
-    UserDetails,
-    // We can provide custom detail-types to match on
-    "Create" | "Update" | "Delete"
-  > {}
-```
+There is a default Event Bus in every region of an AWS account. It contains events emitted by your AWS Resources, such as when a Step Function execution completes, or when a scheduled trigger fires.
 
-## Filter Events with an EventBusRule
-
-An `EventBusRule` filters events flowing through an `EventBus` using some conditional expression.
+Functionless provides an easy way to work with the default bus.
 
 ```ts
-const onSignUp = bus.when(
-  scope,
-  "OnSignUp",
-  (event) => event["detail-type"] === "SignUp"
-);
+const defaultBus = EventBus.default();
+defaultBus.when((event) => event.source === "lambda");
 ```
 
-See the [Syntax Guide for Event Patterns](./syntax.md#event-patterns) documentation for a detailed reference on the syntax supported within an `EventBusRule`.
+## Adopting CDK Resources
 
-## Transform an Event
-
-The `map` function can be applied to an `EventBusRule` to transform the structure of the event before integration.
+To turn a CDK Event bus into a Functionless EventBus using `EventBus.fromBus`.
 
 ```ts
-const onSignUpTransformed = onSignUp.map((event) => ({
-  type: "SignUp",
-  ...event,
-}));
+const awsBus = new aws_events.EventBus(stack, "awsBus");
+const bus = EventBus.fromBus(awsBus);
 ```
 
-See the [Syntax Guide for Event Transforms](./syntax.md#event-transforms) documentation for a detailed reference on the syntax supported within an `EventBusTransform`.
-
-## Pipe an Event to an Integration
-
-The `pipe` function can be applied to an `EventBusRule` or `EventBusTransform` to send events on to an Integration, such as a Lambda `Function`, `StepFunction` or another `EventBus`.
+This can also be done with imported AWS CDK buses
 
 ```ts
-const processSignUpEvent = new Function(
-  scope,
-  "ProcessSignUp",
-  async (event: OnSignUp) => {
-    await table.putItem({
-      item: event,
-    });
-  }
+const awsBus = new aws_events.EventBus.fromEventBusName(
+  stack,
+  "awsBus",
+  "someBusInTheAccount"
 );
-
-onSignUp.pipe(processSignUpEvent);
-```
-
-## Declare an Event Type
-
-Functionless supports well typed events, lets add our event schema to Typescript.
-
-```ts
-interface UserDetails {
-  id?: string;
-  name: string;
-  age: number;
-  interests: string[];
-}
-
-interface UserEvent
-  extends functionless.EventBusRuleInput<
-    UserDetails,
-    // We can provide custom detail-types to match on
-    "Create" | "Update" | "Delete"
-  > {}
+const bus = EventBus.fromBus(awsBus);
 ```
 
-## Create an EventBusRule
-
-Now that you have a wrapped `EventBus`, lets add some rules.
+## Put Events to your bus from other Resources
 
-Functionless lets you write logic in Typescript on the type safe event.
+Event Bus integrates with other AWS Resources via functionless.
```

Comment:
What does this mean?
> via Functionless

Reply:
Just the authoring integration support.

Reply:
Updated

> Functionless supports `putEvents` integrations with other AWS Resources.

Reply:
I don't understand what that means? Do you mean something like:

> The `putEvents` Integration is supported in Lambda Functions (link), Step Functions (link), Appsync Resolvers (link) and API Gateway Integrations (link)

Overall advice is to be more explicit and thorough with documentation.